### DISCUSSION TITLE
Cannot access the Sqlite database with NextJS when deployed on aws amplify

### DISCUSSION THREAD

**Discussion by Scientist-Ansh** [prisma/prisma#17850] [Q&A]
Using [this](https://github.com/prisma/prisma-examples/tree/latest/typescript/rest-nextjs-api-routes) example app provided in the official prisma-examples works fine locally but when I deploy it on aws amplify, the deployed website is unable to access the database file and the get request errors out with the following error - 

`{"message":"\nInvalid prisma.onboardingResult.findMany()  invocation:\n\n\nError querying the database: unable to open database file: /var/task/node_modules/.prisma/client/./dev.db"}`

**Comment by nurul3101** [comment]
Hello @Scientist-Ansh 👋 

Is there an SQLite database created on AWS Amplify? If yes, can you check if the location of the .db file is correct?
The error message indicates that PrismaClient is unable to locate and query the database at specified location. 

Also, I would recommend you to check if AWS amplify supports hosting SQLite database. You can use other services like EC2 or similar to host an SQLite database.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
